capstone/src/main/java/com/google/sps/servlets/GroupMemberServlet.java

```diff
@@ -26,49 +34,71 @@
   @Override
   // Adds a new member to a group 
   public void doPost(String userId, HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
+
+    String email = request.getParameter("email");
     Long groupId = Long.parseLong(request.getParameter("groupId"));
-    
+
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Entity group = this.getGroupFromId(response, groupId, datastore);
-    if (group != null && !userId.equals("")) {
+    Entity memberEntity = getMemberEntity(email, response, datastore);
+    Entity group = ServletHelper.getEntityFromId(response, groupId, datastore, "Group");
+    if (group == null) return;
+
+    String memResponse = doPost_helper(response, datastore, groupId, memberEntity, group);
```

Comment:
It doesn't look like you are actually using memResponse for anything right now? I want to understand what your intention is here?

Reply:
I was using it for my own documentation, which could eventually be sent as json and displayed to the user to let them know if the user was added, etc.

Reply:
Since you aren't using it atm then just call doPost_helper without assigning it to a variable